apps/charterafrica/src/components/Postcard/PostCard.js

```diff
@@ -0,0 +1,113 @@
+import { Link } from "@commons-ui/next";
+import { CardActionArea, CardContent, CardMedia, styled } from "@mui/material";
+import PropTypes from "prop-types";
+import { forwardRef } from "react";
+
+import LineClampedRichTypography from "../LineClampedRichTypography";
+
+import { neutral } from "@/charterafrica/colors";
+
+const Card = styled(CardActionArea)(({ ownerState, theme }) => ({
+  backgroundColor: "#fff",
+  transition: theme.transitions.create(["filter", "border-color"]),
+  width: "100%",
+  "&:hover": {
+    ...(ownerState.variant === "outlined" && {
+      borderColor: ownerState.href ? neutral[700] : "transparent",
+      filter: "drop-shadow(0px 12px 24px rgba(0, 0, 0, 0.1))",
+      backgroundColor: "#fff",
```

Comment:
Don't see any of these in the hover effect defined [here](https://www.figma.com/file/gbnaO1FYPq16V2gAFbLOkl/CfA---EPD-Charter-Africa---Sep-2022?node-id=7395%3A1569&t=MSv29AifG9KxxtVZ-0).